modules/oe_theme_content_publication/src/Plugin/PageHeaderMetadata/PublicationContentType.php

```diff
@@ -75,13 +75,10 @@ public function applies(): bool {
    * {@inheritdoc}
    */
   public function getMetadata(): array {
-    $metadata = parent::getMetadata();
-
     $node = $this->getNode();
-    $timestamp = $node->get('oe_publication_date')->date->getTimestamp();
-    $metadata['metas'] = [
-      $this->dateFormatter->format($timestamp, 'oe_theme_publication_date'),
-    ];
+
+    $metadata = parent::getMetadata();
+    $metadata['metas'][] = $this->getSeparatedSkosMeta($node->get('oe_publication_type'), ' | ');
```

Comment:
Aren't the metas already formatted by the template with a pipe separator? ` | `

Reply:
Also, I once discussed this with @kugta about separating the same type of things with a comma, instead of a pipe, could you maybe check with him if he wants to do that in this case too? Meaning, if it's ok for him to use a comma instead of a pipe in this case.

Reply:
Yes, template use pipe as a separator for meta. News header uses comma as a separator in another use case. I've rolled changes in news back.